public/js/all.js

```diff
@@ -533,23 +534,21 @@ $(function () {
   repeatFade($('.trophyimageincomplete'), 200, 300);
 });
 
-function removeComment(artID, commentID) {
+function removeComment(artTypeID, artID, commentID) {
   if (!window.confirm('Ary you sure you want to permanently delete this comment?')) {
     return false;
   }
 
   var posting = $.post('/request/comment/delete.php', { a: artID, c: commentID });
-  posting.done(onRemoveComment);
+  posting.done(function onRemoveComment(data) {
```

Comment:
```suggestion
  posting.done(function(data) {
```

Reply:
Was modelled after another recent change: https://github.com/RetroAchievements/RAWeb/pull/927/files#diff-f6d8800068b5d78d8a744d9f951adc97ee8738c9cda6636c3ccea921f059cce5R53

Corrections made.

Reply:
I missed that detail in the other PR - thanks 🙏